Game/Game.Test.js

```diff
@@ -19,96 +16,186 @@ describe('Game', () => {
             })
+
         })
 
         describe('reviewAttempt() method (this.correctGuesses)', () => {
             it("10. a bad guess should not increase the length of `this.correctGuesses`", () => {
-    
+
                 const a = new Game('game', 3);
-    
+
                 a.reviewAttempt('c');
-    
+
                 expect(a.correctGuesses.length).to.equal(0);
             })
 
             it("11. a correct guess should increase the length of `this.correctGuesses`", () => {
-    
+
                 const a = new Game('game', 3);
-    
+
                 a.reviewAttempt('g');
-    
-                expect(a.correctGuesses.length).to.equal(1);
+
+                expect(a.correctGuesses).to.deep.equal(['g']);
             })
 
             it("12. a correct guess should be the last index of `this.correctGuesses`", () => {
-    
+
                 const a = new Game('game', 3);
-    
+
                 a.reviewAttempt('e');
                 a.reviewAttempt('g');
-    
-                expect(a.correctGuesses.slice(-1)[0]).to.equal('g');
+
+                expect(a.correctGuesses).to.deep.equal(['e','g']);
             })
+
+            it("13. when all letters have been guessed, it should end in a win", () => {
+
+              const a = new Game('game', 3);
+
+              a.reviewAttempt('a');
+              a.reviewAttempt('e');
+              a.reviewAttempt('g');
+
+              expect(a.reviewAttempt('m')).to.equal('The man is alive');
+              expect(a.correctGuesses).to.deep.equal(['a','e','g','m']);
+            })
+        })
+
+        describe('displayRevealed() method (this.correctGuesses)', () => {
+
+          it('14. to start, it should return one `_` per letter', () => {
+            const a = new Game('game', 3);
+
+            expect(a.displayRevealed()).to.deep.equal(['_','_','_','_']);
+          })
+
+          it('14. for each correct guess, it include those letters', () => {
+            const a = new Game('game guru', 3);
+
+            expect(a.displayRevealed()).to.deep.equal(
+              ['_','_','_','_',' ','_','_','_','_']
+            );
+
+            a.reviewAttempt('a');
+            expect(a.displayRevealed()).to.deep.equal(
+              ['_','a','_','_',' ','_','_','_','_']
+            );
+
+            a.reviewAttempt('g');
+            expect(a.displayRevealed()).to.deep.equal(
+              ['g','a','_','_',' ','g','_','_','_']
+            );
+
+            a.reviewAttempt('u');
+            expect(a.displayRevealed()).to.deep.equal(
+              ['g','a','_','_',' ','g','u','_','u']
+            );
+
+            a.reviewAttempt('e');
+            expect(a.displayRevealed()).to.deep.equal(
+              ['g','a','_','e',' ','g','u','_','u']
+            );
+
+            a.reviewAttempt('m');
+            a.reviewAttempt('r');
+            expect(a.displayRevealed()).to.deep.equal(
+              ['g','a','m','e',' ','g','u','r','u']
+            );
```

Comment:
These are some LIT tests. Do we have the ability to input multiple words? Because that would be super cool! Also, let's keep in mind a letter limit.